presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -692,6 +693,11 @@ public SystemSessionProperties(
                         "Force single node output",
                         featuresConfig.isForceSingleNodeOutput(),
                         true),
+                booleanProperty(
+                        FORCE_COORDINATOR_EXECUTION,
+                        "force run on coordinator",
+                        false,
+                        false),
```

Comment:
nit: Usually we add a configuration property along with a session property

Reply:
i have add